### PR TITLE
React tutorial: update asset example, code updates

### PR DESCRIPTION
**Pull request by kremalicious**
- follow newest changes in OEP-8
- update feedback
- simplify setup

### CODE REVIEW

**Comment by ttmc** [comment]
Thanks, I was wondering about this. I should be able to test it tomorrow.

**Comment by kremalicious** [comment]
So basically the tutorial was broken because the search returns nothing, but the search result is needed to get into consume flow. Submitted fix to squid-js https://github.com/oceanprotocol/squid-js/pull/203

After that is released, I can test further too. So far registering works nicely.

File: content/tutorials/react-setup.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ description: This tutorial shows how you can build a basic [React](https://react
 
 - `Node.js` >= 10 is installed. You can check using `node -v`
 - `npm` >= 5.2 is installed. You can check using `npm -v`
-- Git clone the [oceanprotocol/barge](https://github.com/oceanprotocol/barge) repository, then in that directory:
+- `Spree` local Ocean test hetwork
```
Comment (ttmc) [inline content/tutorials/react-setup.md]: "hetwork" should be "network"

File: content/tutorials/react-setup.md [hunk 2]
```diff
@@ -96,7 +97,7 @@ async componentDidMount() {
     parityUri: "http://localhost:8545",
     secretStoreUri: "http://localhost:12001"
   })
-  console.log("Finished loading contracts!")
+  console.log("Finished loading contracts.")
```
Comment (ttmc) [inline content/tutorials/react-setup.md]: If you're going to change that here then you should change it in all the other places too.

File: content/tutorials/react-setup.md (hunk 1 above)
Comment (kremalicious) [reply content/tutorials/react-setup.md]: thanks, changed!

File: content/tutorials/react-setup.md (hunk 2 above)
Comment (kremalicious) [reply content/tutorials/react-setup.md]: I actually did everywhere I think

**Comment by kremalicious** [comment]
ok, now search works again with squid-js 0.5.6 and I adapted for new search response in this PR.

Now upon consuming I also get `Requested did is not found in the keeper network`

So we're simply searching for `10 Monkey Species Small` and this returns 21 results in my test cause I registered the same asset over and over again. So there are multiple asses under that name in my local Spree network, but they all have different DIDs. Still, all of them should be actually there so all of them should be found...

**Comment by ttmc** [comment]
I like all the changes you made to the text, and the idea of moving the asset into its own file.

**Pull request merged**